src/main/java/pl/betoncraft/betonquest/compatibility/mythicmobs/MythicMobKillObjective.java

```diff
@@ -38,21 +38,22 @@
  */
 public class MythicMobKillObjective extends Objective implements Listener {
 
-	private final String name;
+	private final List<String> names = new ArrayList<>();
```

Comment:
There's no import for `List`.

Reply:
And if you already changing something I'dd recommend using a HashSet here to slightly increase the performance.

Reply:
A damn, surely something has to go wrong without IDE^^ 

Good catch with the hashset, even so its premature optimization and with lists as small as this would most likely not be that much off a performancegain, if at all :)
Has anybody tested this to be the case? Will google it later :3

Reply:
I wouldn't call it premature, more good style since you'd use the list as a `Set` anyways - you don't care about the order of the items, just the items themselves.

Reply:
Well it would be if the only reason for it are performance reasons without measuring a negativ performance impact. :)